aya/src/maps/mod.rs

```diff
@@ -153,6 +167,31 @@ impl Map {
         Ok(fd)
     }
 
+    pub fn from_pinned<P: AsRef<Path>>(&mut self, path: P) -> Result<RawFd, MapError> {
```

Comment:
I think this shouldn't be pub for now.

At some point I think we'll want to expose something like `let map = Map::from_pinned(path)` to load a pinned map before going through the loader (and without having a `bpf_map_def` for the map).

Reply:
ack, made it `pub(crate)` instead